go.mod

```
@@ -2,6 +2,6 @@ module github.com/minio/rsync-go
 
 go 1.17
```

How about:
```suggestion
go 1.22
```

In the fuzzing test, `math/rand/v2` is imported which is part of go1.22 release.
`go test` / `go vet` are broken when executed on go1.17 (in the current state of this PR), it should be still importable.
Updating a Golang project version is quite easy.

On the other hand, we can downgrade to `math/rand`.

---

https://github.com/minio/rsync-go/blob/8dc6f97e7eef301a78c9e5b827255a7c44f76eac/.github/workflows/vulncheck.yml#L17
needs to be updated too